ext/ddtrace_profiling_native_extension/extconf.rb

```diff
@@ -5,14 +5,26 @@
 
 require_relative 'native_extension_helpers'
 
-def skip_building_extension!
+SKIPPED_REASON_FILE = "#{__dir__}/skipped_reason.txt".freeze
+begin
+  File.delete(SKIPPED_REASON_FILE)
+rescue StandardError => _e
+  # Not a problem if the file doesn't exist or we can't delete it
+end
```

Comment:
Seem like an appropriate use for a traditionally "bad" pattern:
```suggestion
# Not a problem if the file doesn't exist or we can't delete it
File.delete(SKIPPED_REASON_FILE) rescue nil
```

Not sure our friend Rubocop will let this slide (or make it uglier with its `# disable` statements)